.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -31,22 +31,10 @@ Steps to reproduce the behavior:
 
 **Version information**
 <!-- please fill out this information below completely, it will help us solve your issue faster -->
-<!-- if you are trying to build go-filecoin -->
-- Go: <!-- go version -->
+- Filecoin: <!-- go-filecoin inspect all -->
```

Comment:
Do you want people to paste all of the `inspect` contents, or just the version? If the former, maybe:

`<!-- Run `go-filecoin inspect all` and paste full contents here-->`

Reply:
Good point, I would prefer all the contents. Will rename this to `Diagnostic information`, and also address the sentence structure.